app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
```

Comment:
- [ ] We can not log in because it doesn't find the ID user.

![Capture web_23-12-2021_13227_localhost](https://user-images.githubusercontent.com/61562626/147170098-3bb05154-6caa-4e17-b548-f22df8ff67c6.jpeg)